rero_ils/modules/items/mappings/v7/items/item-v0.0.1.json

```diff
@@ -48,6 +48,9 @@
       "second_call_number": {
         "type": "keyword"
       },
+      "enumeration_and_chronology": {
```

Comment:
'enumerationAndChronology' instead of `enumeration_and_chronology` ?

Reply:
right ! Done